### DISCUSSION TITLE
Silly idea: Retro mode

### DISCUSSION THREAD

**Discussion by Petarkco** [f1multiviewer/issue-tracker#234] [Feature requests] [answered]
Just a silly suggestion but maybe add an option to display live timing data like it's 1999 with the yellow squares for position etc.

Or the F1 Digital speedo from the 2000s:
https://www.youtube.com/watch?v=rilHFWKnUw8

**Comment by f1multiviewer** [comment, marked as answer]
I think you're pretty far by now with your integration to offer this, so I'm gonna say this is solved 👍
